src/checks_test.cc

```diff
@@ -306,5 +306,52 @@ TEST(LinterTest, CheckImports) {
             "Imports should specify the type 'MODULE' or 'PROTO'.");
 }
 
+TEST(LinterTest, CheckExpressionParantheses) {
+  LinterOptions option;
+
+  EXPECT_EQ(CheckExpressionParantheses(
+                "SELECT D where a AND b*2 OR (c AND x OR T);", option)
+                .GetErrors()
+                .size(),
+            2);
+
+  EXPECT_TRUE(
+      CheckExpressionParantheses("SELECT D where a or b or c;", option).ok());
+
+  EXPECT_TRUE(
+      CheckExpressionParantheses("SELECT D where a and b and c;", option).ok());
+  EXPECT_FALSE(
+      CheckExpressionParantheses("SELECT D where a and b or c;", option).ok());
+  EXPECT_FALSE(
+      CheckExpressionParantheses("SELECT D where a or b and c;", option).ok());
+  EXPECT_TRUE(CheckExpressionParantheses(
+                  "SELECT D where (       a or b   ) and c;", option)
+                  .ok());
+}
+
+TEST(LinterTest, CheckCountStar) {
+  LinterOptions option;
+  EXPECT_TRUE(CheckCountStar("SELECT COUNT", option).ok());
+  EXPECT_TRUE(CheckCountStar("SELECT COUNT(0)", option).ok());
+  EXPECT_FALSE(CheckCountStar("SELECT COUNT(1)", option).ok());
+  EXPECT_TRUE(CheckCountStar("SELECT COUNT(*)", option).ok());
+}
+
+TEST(LinterTest, CheckKeywordNamedIdentifier) {
+  LinterOptions option;
+  LinterResult result = CheckKeywordNamedIdentifier("SELECT Date", option);
+  result.PrintResult();
```

Comment:
nit: good practice is to make tests print stuff only if there is an error. This way, you can quickly see in the test output the error log from the failing test.

Reply:
Yes, I erased it. (forgot that in there.)